code-challenges.test.js

```diff
@@ -32,28 +41,106 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+
+describe("divisibleBy", () => {
+    it ("takes in a number and checks if it is divisible by three", () => {
+        expect(divisibleBy(object1)).toEqual((divisibleBy)("0 is divisible by three"));
+        expect(divisibleBy(object2)).toEqual((divisibleBy)("-7 is not divisible by three"))
+        expect(divisibleBy(object3)).toEqual((divisibleBy)("15 is divisible by three"))
+    })
+})
+
+
+// // b) Create the function that makes the test pass.
+
+// // process: declare a function named divisible by that takes in a number, and provides a conditional in which if the number is divisible by 3 it returns "this number is divisible by 3", and " this number is not divisible by three" if not
+
+
+const divisibleBy = (num) => {
+   if (num % 3 === 0) {
+    return "0 is divided by three" }
+    else if (num % 3 == 1) {
```

Comment:
This is so close! This problem is asking for 1 of 2 returns. It only wants to know if (num) is evenly divisible by 3. If it is not than it can return that is was not evenly divisible. I recommend trying to look at this dynamically and making the returns less hard coded.